oracle/assets/dashboards/oracle_overview.json

```diff
@@ -0,0 +1,900 @@
+{
+    "board_bgtype": "board_graph",
+    "board_title": "Oracle",
```

Comment:
(nit) 

```suggestion
    "board_title": "Oracle Overview",
```

Reply:
This I can't change currently without some extra effort outside this repo  :) This is an existing payload.